### DISCUSSION TITLE
Connecting DBeaver with Mysql

### DISCUSSION THREAD

**Discussion by wexelmayl57** [dbeaver/dbeaver#10778] [Q&A] [answered]
I have a mac book and can't use the client my class uses (heidisql) so I tried dbeaver but I have trouble with connecting a database to dbeaver. Do you guys have any suggestions on how to proceed?

**Comment by flambchap** [comment, marked as answer]
You can try [this guide](https://github.com/dbeaver/dbeaver/wiki/Create-Connection).  Be aware, though, that we have three MySQL drivers: 

* MySQL, for MySQL 8.0.0+
* MySQL 5, for MySQL 5.x.x
* MySQL (old), for antique MySQL's

Be sure to choose the right one.
